libs/sdk-ui-kit/src/List/ListItem.tsx

```diff
@@ -125,61 +179,14 @@ export class SingleSelectListItem extends Component<ISingleSelectListItemProps,
                 </BubbleHoverTrigger>
             );
         }
-
         return titleElement;
     };
 
-    private renderIcon = (icon: string | ReactNode) => {
-        if (icon && typeof icon === "string") {
-            const iconClasses = cx("gd-list-icon", icon);
-            return <span role="icon" aria-hidden={true} className={iconClasses} />;
-        }
-        if (icon) {
-            const iconClasses = cx("gd-list-icon");
-            return (
-                <span role="icon" aria-hidden={true} className={iconClasses}>
-                    {icon}
-                </span>
-            );
-        }
-        return null;
-    };
-
-    private renderSeparatorItem = () => {
-        return (
-            <div
-                role="item-separator"
-                className={cx(
-                    "gd-list-item",
-                    "gd-list-item-separator",
-                    "s-list-separator",
-                    this.props.className,
-                )}
-            />
-        );
-    };
+    const renderInfo = () => {
+        if (!info) return null;
```

Comment:
can you please wrap it in `{ }`? I'm surprised these inlines are not removed by prettier and caught by eslint.
I see this pattern multiple times in the file.